app/Community/RouteServiceProvider.php

```diff
@@ -366,6 +366,9 @@ protected function mapWebRoutes(): void
                     Route::put('password', [UserSettingsController::class, 'updatePassword'])->name('api.settings.password.update');
                     Route::put('email', [UserSettingsController::class, 'updateEmail'])->name('api.settings.email.update');
 
+                    Route::post('username-change-request', [UserSettingsController::class, 'storeUsernameChangeRequest'])
+                        ->name('api.settings.username-change-request.store');
```

Comment:
Validation errors aren't reported correctly.
![image](https://github.com/user-attachments/assets/b99aef45-196c-4702-b0e1-c422e193193c)

Reply:
I'm having trouble reproducing this:

![Screenshot 2025-01-19 at 11 06 13 AM](https://github.com/user-attachments/assets/04eef2da-b88d-45d9-a6ec-5dcd3e601ff2)

I did make one minor change to client-side validation in latest (users cannot request a change to the name they are currently set to). But, even before this change, I was seeing server-side validation messages appear in the toast.

Reply:
Try with invalid characters. I think I was using "It's-a-me, Mario" as my display name.

Reply:
That worked, thanks.

I've made an update in latest, but it's client-side. I think for these particular requests, it may be best to catch them before they get sent to the server:

![Screenshot 2025-01-19 at 12 04 51 PM](https://github.com/user-attachments/assets/98d16a04-a832-4a1e-b54e-588a34bfad5c)

Reply:
Suggest changing message to match sign up.
![image](https://github.com/user-attachments/assets/68a01f31-fe41-4b00-b0e9-db9bee6c7b08)

vs
![image](https://github.com/user-attachments/assets/abd40d8f-6587-4f63-9eda-c88781076591)

Reply:
Updated these client-side validation messages in latest.